spec.bs

```diff
@@ -107,18 +108,29 @@ dictionary AuctionAdInterestGroup {
 
 <div algorithm="joinAdInterestGroup()">
 
-The <dfn for=Navigator method>joinAdInterestGroup(|group|, |durationSeconds|)</dfn> method steps
-are:
+The <dfn for=Navigator method>joinAdInterestGroup(|group|)</dfn> method steps are:
+<div class="note">
+
+Note: Temporarily, Chromium does not include the <span class="allow-2119">required</span> keyword

```

Comment:
Oops, this produced a nested note block, which we can remove by taking out the "Note:".
```suggestion
Temporarily, Chromium does not include the <span class="allow-2119">[=dictionary member/required=]</span> keyword
```

Also, hah @ the 2119 error. It should probably either be monospace or this link to its meaning, so it doesn't look like we've omitted a keyword that we were required to include.

Reply:
monospace didn't work. 
<span class="allow-2119">`required`</span> gave the same 2119 lint error.

Reply:
Yes, it'll still need to be surrounded by the `<span class="allow-2119">` to satisfy Bikeshed, but it should also have some typography to help people reading the spec.

Reply:
yeah I mean surrounding `required` with `<span class="allow-2119">` still gives the 2119 error. I had to remove the backticks (i.e., only keep the string requried without the backticks) to get rid of the error. Not sure if it's a bug.

Reply:
I think there is a bug here, of some sort. I'm not sure exactly which strings should be allowed, but more than are allowed: https://github.com/speced/bikeshed/issues/2569.